assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -13,7 +13,8 @@ export const tests: Tests = {
       },
     },
     isActive: true,
-    independence: 3,
+    independent: true,
```

Comment:
Can you add a test where independent is false?

Reply:
Good call!